src/Umbraco.Core/Runtime/CoreRuntime.cs

```diff
@@ -288,6 +299,17 @@ private void DoUnattendedInstall(IUmbracoDatabaseFactory databaseFactory)
             }
         }
 
+        private void DoUnattendedUpgrade(DatabaseBuilder databaseBuilder)
+        {
+            var plan = new UmbracoPlan();
+            Logger.Info<CoreRuntime>("Starting unattended upgrade.");
```

Comment:
I changed this to `using (ProfilingLogger.TraceDuration<CoreRuntime>("Starting unattended upgrade.", "Unattended upgrade completed."))` to profile the upgrade since it provides a little more info.